Dockerfile

```diff
@@ -17,6 +17,12 @@ COPY ./MANIFEST.in ./MANIFEST.in
 COPY ./requirements.txt ./requirements.txt
 COPY ./setup.py ./setup.py
 
+ARG YTT_VERSION="0.30.0"
```

Comment:
if this is just for the test, I don't think we need it permanently, do we?

Reply:
Correct. Just for the test. Def something we don’t need permanently.